recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,5 @@
+rem packages for nbconvert
```

Comment:
I'm aware that `rem` is a way to put a comment in bat scripts. I wrote the original recipe mentioned by OP.

I simly read the comment which states "packages for nbconvert", as if the latex distro strictly requires it, whereas I've worked without it for more than a year without issues.

My question is therefore simple; why do users of this package need nbconvert/these additional packages to be preinstalled? They seem to related to the application OP has in mind but unrelated to the latex distro repackaging on its own.